README.md

```diff
@@ -106,12 +81,12 @@ This will bring up an editor. Type build args into that file like this:
 ```
 enable_castanets=true
 enable_nacl=false
+is_debug=false
```

Comment:
Sad...:( But we're going to turn on DCHECK and then do debug build as soon as we can.